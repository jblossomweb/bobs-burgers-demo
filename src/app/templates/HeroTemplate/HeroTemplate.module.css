.container {
  @apply text-center;
}

.header {
  @apply min-h-screen
    flex
    flex-col
    items-center
    justify-center
    animate-fadein;

  font-size: calc(8px + 2vmin);
}
