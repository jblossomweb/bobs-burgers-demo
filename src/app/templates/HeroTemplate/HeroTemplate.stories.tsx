import React from 'react'
import type { ComponentStory, ComponentMeta } from '@storybook/react'

import HeroTemplate, { type Props } from './HeroTemplate'

export default {
  title: 'templates/HeroTemplate',
  component: HeroTemplate,
  argTypes: {
    title: { control: 'text' },
    children: { control: 'text' },
  },
} as ComponentMeta<typeof HeroTemplate>

const Template: ComponentStory<typeof HeroTemplate> = ({
  title,
  children,
}: Props) => <HeroTemplate title={title}>{children}</HeroTemplate>

export const Sample = Template.bind({})
Sample.args = {
  title: 'Sample',
  children: 'Page content',
}
