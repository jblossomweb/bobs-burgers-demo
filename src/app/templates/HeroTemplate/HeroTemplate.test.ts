import storySnaps from 'test/utils/storySnaps'
import * as stories from './HeroTemplate.stories'

describe('app/templates/HeroTemplate', () => {
  describe('stories', () => {
    storySnaps(stories)
  })
})
