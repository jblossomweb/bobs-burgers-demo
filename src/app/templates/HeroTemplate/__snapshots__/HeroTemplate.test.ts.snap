// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app/templates/HeroTemplate stories Sample matches markup snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <header
      class="header"
    >
      Page content
    </header>
  </div>
</DocumentFragment>
`;
